frontend/dockerfile/docs/reference.md

```diff
@@ -1509,11 +1455,52 @@ attempted to be used instead.
 > guide – Leverage build cache](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache)
 > for more information.
 
-## COPY --link
+### COPY --chown --chmod
 
 > **Note**
 >
-> Added in [`docker/dockerfile:1.4`](#syntax)
+> Only octal notation is currently supported. Non-octal support is tracked in
+> [moby/buildkit#1951](https://github.com/moby/buildkit/issues/1951).
+
+```dockerfile
+COPY [--chown=<user>:<group>] [--chmod=<perms> ...] <src> ... <dest>
+```
+
+The `--chown` and `--chmod` features are only supported on Dockerfiles used to build Linux containers,
+and doesn't work on Windows containers. Since user and group ownership concepts do
+not translate between Linux and Windows, the use of `/etc/passwd` and `/etc/group` for
+translating user and group names to IDs restricts this feature to only be viable for
+Linux OS-based containers.
+
+All new files and directories are created with a UID and GID of 0, unless the
```

Comment:
This is only true for files copied from the build context. Otherwise, files keep their original uid/gid. Also, this conversion happens in build context transfer so is not `COPY`-specific. Eg. if mounts are used then it also build context files have already become uid/gid=0 before.

Reply:
OK. Since this isn't new or changed in this PR (I just moved it into a section) I think we can do a follow-up. I think we can call this out on the context page and link to there. https://docs.docker.com/build/building/context/